DemoApp/DemoApp/Resources/Styles/CustomThemeDark.xaml

```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<ResourceDictionary xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             x:Class="DemoApp.Resources.Styles.CustomThemeDark">
+  <Color x:Key="Blue">red</Color>
```

Comment:
User shouldn't be able to change the 'color' names. Instead they should be ony able to change the namings - like 'PrimaryButtonBackgroundColor', 'PrimaryButtonTextColor'...

Reply:
Currently, we have color names like blue, red, etc., in both modes. After adding more colors we can identify which are duplicated and naming for each color.